CONTRIBUTING.md

```diff
@@ -0,0 +1,62 @@
+## Coding style
+
+* HOL source code
+  - Spaces, no tabs
+  - No unicode
+  - `snake_case` (e.g. `bir_number_of_mem_splits_def`)
+
+
+
+## Branch policy
+
+### tags
+
+tags should have as many **completed features** as possible:
+ - no cheat
+ - must correctly compile
+ - self tests must succeed
+ - code should be tested
+
+Follow these instructions whenever you merge to master:
+  - `grep` for "cheat"
+  - check that the `README` is up to date (especially tool status)
+  - find a reviewer for your Pull Request
+
+### `master` branch
```

Comment:
This title should be before `Follow these instructions whenever you merge to master:`

Reply:
yes